src/app/application.cpp

```diff
@@ -405,15 +405,62 @@ void Application::processParams(const QStringList &params)
         return;
     }
 #endif
+    BitTorrent::AddTorrentParams torrentParams;
+    TriStateBool skipTorrentDialog = TriStateBool::Undefined;
 
     foreach (QString param, params) {
         param = param.trimmed();
+
+        // Process strings indicating options specified by the user.
+        
+        if (param.startsWith("@savePath=")) {
+            torrentParams.savePath = param.mid(10);
+            continue;
```

Comment:
You are saying this as if everybody likes to read a loop body to the end to check whether there is anything after a sheet of `if`s.